src/qz/ui/component/CertificateTable.java

```diff
@@ -126,7 +170,22 @@ public Component getTableCellRendererComponent(JTable table, Object value, boole
 
             // First Column
             if (value instanceof CertificateField) {
-                label = stylizeLabel(STATUS_NORMAL, label, isSelected);
```

Comment:
@Vzor- do you mind explaining these changes?  At a glance I would not expect the first column logic to change as a result of this PR.  Perhaps it was due to some unreverted changes, but I would expect this logic to be largely untouched.